srcpkgs/python3-pmw/template

```diff
@@ -0,0 +1,19 @@
+# Template for python-pmw
+pkgname=python3-pmw
+version=2.0.1
+revision=1
+noarch=yes
+short_desc="Python 2 Tkinter widget toolkit"
+homepage="http://pmw.sourceforge.net/"
+license="MIT"
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+wrksrc=Pmw-${version}
+build_style=python3-module
+hostmakedepends="python3-devel tk-devel"
+depends="tk python"
```

Comment:
shoudln't it be python3 instead of python?

Reply:
Ditto.